{"CVE Code": "CVE-2021-27213", "Description": "config.py in pystemon before 2021-02-13 allows code execution via YAML deserialization because SafeLoader and safe_load are not used.", "NVD Link": "https://nvd.nist.gov/vuln/detail/CVE-2021-27213", "Vulnerability Categories": "N/A", "Affected Products": "\nPystemon Project\u00bbPystemonVersions  before (<) 2021-02-13cpe:2.3:a:pystemon_project:pystemon:*:*:*:*:*:*:*:*Matching versions", "EPSS Score": "1.56% Probability of exploitation activity in the next 30 days\n~ 87 % Percentile, the proportion of vulnerabilities that are scored at or less", "EPSS History": "\n #  Date  Old EPSS Score  New EPSS Score  Delta (New - Old) \n 1  2024-12-17  0.84%  1.56%  +0.73 \n 2  2024-07-04  0.85%  0.84%  -0.02 \n 3  2024-06-19  0.69%  0.85%  +0.16 \n 4  2024-01-20  0.66%  0.69%  +0.04 \n 5  2024-01-06  0.47%  0.66%  +0.18 \n 6  2023-10-30  0.56%  0.47%  -0.09 \n 7  2023-09-20  0.48%  0.56%  +0.09 \n 8  2023-03-07  2.28%  0.48%  -1.80 ", "CVSS Scores": "7.5", "CWE": [{"CWE_ID": "CWE-502 Deserialization of Untrusted Data", "CWE_link": "/cwe-details/502/Deserialization-of-Untrusted-Data.html"}], "Reference": [{"ref_link": "https://github.com/cvandeplas/pystemon/commit/dbeb87afefdb63de2f4cff69b6f10c5965d14b54", "ref_desc": "Fixed code execution bug using SafeLoader() \u00b7 cvandeplas/pystemon@dbeb87a \u00b7 GitHubPatch;Third Party Advisory", "ref_cve_link": "N/A"}, {"ref_link": "https://www.huntr.dev/bounties/1-other-pystemon/", "ref_desc": "Arbitrary Code Execution in pystemon | huntrExploit;Third Party Advisory", "ref_cve_link": "N/A"}]}